app/services/arcgis_api/mock/fixtures.rb

```diff
@@ -28,7 +28,7 @@ def self.request_candidates_error
       def self.load_response_fixture(filename)
         path = File.join(
           File.dirname(__FILE__),
-          '../fixtures/arcgis_api_responses',
+          'fixtures',
           filename,
         )
```

Comment:
I am guessing that we moved the fixtures into the `app` dir so that the live app wouldn't open the `spec` dir? I would personally prefer the opposite, I'd leave the fixtures and examples in `spec/` and have the mock service read from there if needed.

Reply:
At the end of the day, it's just files under source control. Classes in `spec/` won't get autoloaded, but the fixtures themselves that are loaded directly will be accessible.

Reply:
Nope I am not concerned. Can always branch deploy test this in `joy` before merging if you're worried!